Mathlib/NumberTheory/NumberField/CanonicalEmbedding.lean

```diff
@@ -858,19 +980,86 @@ theorem exists_ne_zero_mem_ideal_lt (h : minkowskiBound K I < volume (convexBody
     change Countable (span ℤ (Set.range (fractionalIdealLatticeBasis K I)) : Set (E K))
     infer_instance
   obtain ⟨⟨x, hx⟩, h_nz, h_mem⟩ := exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure
-    h_fund (convexBodyLT_symmetric K f) (convexBodyLT_convex K f) h
+    h_fund (convexBodyLT_neg_mem K f) (convexBodyLT_convex K f) h
   rw [mem_toAddSubgroup, mem_span_fractionalIdealLatticeBasis] at hx
   obtain ⟨a, ha, rfl⟩ := hx
   exact ⟨a, ha, by simpa using h_nz, (convexBodyLT_mem K f).mp h_mem⟩
 
-/- TODO: Remove!. Necessary to prevent a timeout that ends at here. #10131 -/
-attribute [-instance] FractionalIdeal.commSemiring in
+/-- A version of `exists_ne_zero_mem_ideal_lt` where the absolute value of the real part of `x` is
```

Comment:
I don't understand the docstring, there is no `x` in the theorem.

Reply:
Indeed. It should be `a`. 

Thanks for the review!